addons.go

```diff
@@ -1,18 +1,24 @@
 package scalingo
 
-import "gopkg.in/errgo.v1"
+import (
+	"github.com/Scalingo/go-scalingo/http"
+
+	"gopkg.in/errgo.v1"
+)
 
 type AddonsService interface {
 	AddonsList(app string) ([]*Addon, error)
 	AddonProvision(app, addon, planID string) (AddonRes, error)
 	AddonDestroy(app, addonID string) error
 	AddonUpgrade(app, addonID, planID string) (AddonRes, error)
+	AddonToken(app, addonID string) (string, error)
 }
 
 var _ AddonsService = (*Client)(nil)
 
 type Addon struct {
 	ID              string         `json:"id"`
+	UUID            string         `json:"uuid"`
```

Comment:
The API is not returning the UUID as ID?

Reply:
Nop

Reply:
Wouldn't it be the way to go?

Reply:
Maybe but it's not really in the scope of this PR.

Reply:
But adding it now to remove it later (and breaking the API) is not the greatest thing to do no?

Reply:
https://github.com/Scalingo/api/pull/1342